scripts/actions/release-notes.py

```diff
@@ -0,0 +1,32 @@
+from github import Github
+from invoke import run
+import os
+import re
+
+# Get token from Workflow environment variable
+token = os.getenv('GITHUB_TOKEN', '...')
+
+# Initialize GitHub with token
+g = Github(token)
+
+# Create heading for Release Notes
+result = "## :rocket: What's new?\n\n\n"
+
+# Get the Docs repo
+repo = g.get_repo("newrelic/docs-website")
+
+# Compare diff between main and develop
+diff = repo.compare("main", "develop")
+
+# Loop through commits and add details to result
+for com in diff.commits:
+  line = "@" + com.author.login
+  line += " - " + com.commit.message.splitlines()[0]
+  line += " [->](" + com.html_url + ")"
+  result += line
+  result += "\n"
```

Comment:
i would be in favor of using f-strings here. i think that would make it easier to understand some of these lines. can even reduce the total number of lines.
example:
```py
line += f"[->]({com.html_url})"
```

Reply:
Excellent i\ll change that now

Reply:
actually, sorry to make another request for change, haha. with all the lines, and variables, i think i would prefer something like:
```py
  result += "@{author} - {message} [->]({url})\n".format(
    author=com.author.login,
    message=com.commit.message.splitlines()[0],
    url=com.html._url
  )
```
i think that is probably an improvement in readability over just using f-strings, since the expected output is clearer.